docs/get-started/build-guide.md

```diff
@@ -31,7 +31,8 @@ Please set them via `--`, e.g. `--build_type=Release`.
 | velox_home             | Specify your own Velox source path to build.                               | ""      |
 | build_velox_tests      | Build Velox tests.                                                         | OFF     |
 | build_velox_benchmarks | Build Velox benchmarks (velox_tests and connectors will be disabled if ON) | OFF     |
-| compile_arrow_java     | Compile arrow java for gluten build to use to fix invalid pointer issues.  | ON     |
+| compile_arrow_java     | Compile arrow java for gluten build to use to fix invalid pointer issues.  | ON      |
+| spark_version          | Build for specified version of Spark. `ALL` means build for all versions   | ALL     |
```

Comment:
Can you put the version "like 3.2, 3.3, 3.4,3.5" etc.? In case people use 3.2.2, etc.